packages/contractkit/src/identity/odis/phone-number-identifier.ts

```diff
@@ -11,6 +12,10 @@ import {
   SignMessageResponse,
 } from './query'
 
+// ODIS minimum dollar balance for pepper quota retrieval
+// TODO change this to new ODIS minimum dollar balance once deployed
+export const ODIS_MINIMUM_DOLLAR_BALANCE = 0.1
```

Comment:
As of this morning, this is now 0.01 cUSD or 0.005 CELO. Can you adjust this and add gold?